nautobot/dcim/forms.py

```diff
@@ -908,7 +908,7 @@ class Meta:
 class DeviceTypeBulkEditForm(BootstrapMixin, AddRemoveTagsForm, CustomFieldBulkEditForm):
     pk = forms.ModelMultipleChoiceField(queryset=DeviceType.objects.all(), widget=forms.MultipleHiddenInput())
     manufacturer = DynamicModelChoiceField(queryset=Manufacturer.objects.all(), required=False)
-    u_height = forms.IntegerField(min_value=1, required=False)
+    u_height = forms.IntegerField(required=False)
```

Comment:
No benefit to `min_value=0`?

Reply:
No, because the underlying model field is a `PositiveSmallIntegerField`.

Reply:
Does the UI widget allow negative numbers without setting `min_value=0` here (obviously the model logic will reject it if submitted, I'm just curious what the UI behavior is)?

Reply:
In the add/edit form, no:

<img width="598" alt="image" src="https://user-images.githubusercontent.com/138052/161784273-c8c3c309-d0b2-4c9a-a48f-880c0034a084.png">

In the bulk edit form, it falls back to validation:

<img width="1126" alt="image" src="https://user-images.githubusercontent.com/138052/161784428-e02b8030-8e2c-47de-8ed5-14e3a8956e1f.png">

If I put in `min_value=0` on the bulk edit form, it mirrors the add/edit form: 

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/138052/161784672-bc797882-5989-4c6e-bce1-373eac7320ce.png">

*whine*

Reply:
This latter one is very difficult to test since the input validation on the form field is done client side, blocking form submission.